tensorboard/plugins/pr_curve/summary.py

```diff
@@ -164,6 +168,74 @@ def op(
         description,
         collections)
 
+def pb(tag,
+       labels,
+       predictions,
+       num_thresholds=None,
+       weights=None,
+       display_name=None,
+       description=None):
+  """Creates a PR curves summary protobuf
```

Comment:
For consistency with other `pb` functions: "`Create a PR curves summary protobuf.`" (imperative mood, full stop at end)

Reply:
Done.